### ISSUE TITLE
Crashes when adding video to track

### ISSUE THREAD

**Issue by knockNrod** [OpenShot/openshot-qt#154]
I'm running a VM of Cinnamon Mint 17.3 x64. I have a 4GB memory, 20GB disk, 4 CPUs. The video is recorded in 4K from a Samsung Note 3 to an MP4, and it's roughly 2.1GB in size. When I drag it from the project video to Track 0, OpenShot exits. No crash dialogue, no report whatsoever. Just gone.

Not sure where (if) you record logs. Syslog shows this segfault: 
Apr 13 09:31:39 tenova-virtual-machine kernel: [  302.026392] player[2647]: segfault at 7f67b1fa5e90 ip 00007f68bd9e9ff0 sp 00007f6804ff7c48 error 4 in libc-2.19.so[7f68bd952000+1bb000]

This log shows when I run "tail -f syslog" as I step through this sequence. Let me know if there's more I can do.

**Comment by jonoomph** [comment]
Please download our latest daily installer:

1.  www.openshot.org/download - click '**Other Installers**' link and grab the newest one
2.  Then enable 'Debug Mode (Verbose)' in the Preferences
3.  Quit OpenShot and delete both log files:
  - **/.openshot_qt/openshot_qt.log**
  - **/.openshot_qt/libopenshot.log**
4.  Re-launch OpenShot and trigger the crash as quickly as possible (to keep the log files small)
5.  Attach **both** log files here

Hopefully we can determine from the .log files what is happening (and crashing). Thanks!

**Comment by knockNrod** [comment]
First, thanks for getting back to me. I was expecting an email notification to replies, and certainly wasn't expecting an immediate response, so I was wrong on both accounts. I apologize for my slow response.

I have the PPA. I didn't check before the update, but after the update it says I'm running OpenShot 2.0.7, which is the latest download.

I've deleted openshot_qt.log, but I don't see a libopenshot.log. I've enabled Verbose mode in Debug. I also purged the saved (constantly recovered) project. After the crash, I still don't see a libopenshot.log file, though. Meanwhile, here's the openshot_qt.log file (extension changed to txt for upload).
[openshot-qt.txt](https://github.com/OpenShot/openshot-qt/files/245069/openshot-qt.txt)

**Comment by knockNrod** [comment]
On the off chance that there were substantive differences between Ubuntu and Mint, I removed Openshot installed via the PPA, and downloaded and ran the downloadable appimage version (neat trick, by the way). Same results. Crashes when dragging the clip from Project Files to Track 0, leaves the openshot-qt log file, but nothing for libopenshot. Let me know if I can do more.

**Comment by knockNrod** [comment]
Ok, my first post appears to not have posted? So, take two. I don't seem to have a libopenshot.log file. Here's the openshot_qt.log file, with the extension changed to allow uploads.
[openshot-qt.txt](https://github.com/OpenShot/openshot-qt/files/245110/openshot-qt.txt)

Any chance this is an issue with audio? I ask because I don't know what the audio format is inside of the Note 3 video file.

**Comment by DarrenBarnett** [comment]
Hey guys, 
I've had a similar problem (hence how I found this post), and thought I'd share what I found.

I'm using 64 bit Ubuntu. (Everything worked fine on my 32 bit laptop)

I managed to find that my problem came from the AAC audio codec not being properly installed on my system. I had to purge libavutil52 

_sudo apt-get purge libavutil52_

Then that wanted to remove a load of programs after (including Openshot), so did : 

_sudo apt-get autoremove_

Then started to reinstall what was necessary (in stages)

_sudo apt-get install libavutil-extra-52 libavutil52
sudo apt-get install libav-tools
sudo apt-get install gstreamer1.0-libav_

Now everything works fine and I can edit video.

Hope this helps. 
Daz
